src/components/Card.js

```diff
@@ -0,0 +1,25 @@
+const Card = (props) => {
```

Comment:
You could destructure the props to make it more cleaner
```suggestion
const Card = ({ openSpots, location, img, rating, reviewCount, title }) => {
```